gotfparse/pkg/converter/converter.go

```diff
@@ -392,3 +431,96 @@ func (t *terraformConverter) getPath(b *terraform.Block, parentPath string) stri
 
 	return fmt.Sprintf("%s.%s", parentPath, blockName)
 }
+
+//func getExpressionsFromAttr(expr hcl.Expression) []hcl.Traverser {
```

Comment:
Can we remove the commented out code? If it is saving some context about the future, lets capture it as an issue, and it if is remind someone about the past, we can comment related code.

Reply:
whoops, removed. sorry, this PR got pretty big, and got pretty weird. I may not have looked into the last few commits carefully. thanks for catching this!